api/submit/submit.ts

```diff
@@ -0,0 +1,93 @@
+import type { Handler } from "@netlify/functions"
+import { getUser, saveAnswer, verifyRecaptcha } from "./db"
+
+type RecaptchaResponse = {
+  success?: boolean
+  "error-codes"?: string[]
+}
+
+export const handler: Handler = async event => {
+  // only accept post requests
+  let data = {}
+  if (event.httpMethod !== "POST")
+    return {
+      statusCode: 501,
```

Comment:
405 Method Not Allowed response status code

Reply:
done